htdocs/langs/fr_FR/admin.lang

```diff
@@ -88,6 +88,8 @@ SearchString=Chaîne de recherche
 NotAvailableWhenAjaxDisabled=Non disponible quand Ajax est désactivé
 AllowToSelectProjectFromOtherCompany=Sur les éléments d'un tiers, autorise la sélection d'un projet lié à un autre tiers
 TimesheetPreventAfterFollowingMonths=Empêcher l'enregistrement du temps consacré après le nombre de mois suivant
+PROJECT_DISPLAY_LINKED_BY_CONTACT=Afficher les projets liés par un contact
```

Comment:
@rycks translations need to be in en_US, other are overwrited by transifex sync

Reply:
@frederic34 oooops !